### ISSUE TITLE
Change cache storage location

### ISSUE THREAD

**Issue by ghost** [FreeTubeApp/FreeTube#464]
Is there an ability to change the cache storage location? It currently saves data to the AppData folder, but the hard drive for this is quite small and I would like to minimize unnecessary data storage on this drive.

**Comment by AE720** [comment]
A workaround for when and if they implement that option is to make a symbolic link if you're using windows. You can paste the path you want in quotes and change USERNAME to your actual username and then run it in CMD

mklink /d "Path to desired folder" "C:\Users\USERNAME\AppData\Roaming\FreeTube\Cache"

**Comment by Atrate** [comment]
I would ask for the same to be possible to be done for Linux. ~/.cache is there for a reason.

**Comment by sundaran1122** [comment]
I think you can do the same thing as windows.
``` bash
ln -sf ~/.cache/FreeTube ~/.config/FreeTube
```

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Closing in favor of https://github.com/FreeTubeApp/FreeTube/issues/6454

**Comment by Atrate** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc https://github.com/FreeTubeApp/FreeTube/issues/6454 was closed as wontfix, I don't think this issue should be closed. I don't see *any* reason why the default should *not* be stored properly, according to XDG standards.

**Comment by AE720** [comment]
Yeah, that's kind of lazy, especially considering how trivial it would be to implement. This is the only software I have to do anything hacky for. I've never seen a program that didn't let me change the cache, stremio, google drive, etv

**Comment by absidue** [comment]
@AE720 If you can figure out a way to get Chromium to write it's own cache files to a different directory without having to patch and recompile Chromium, otherwise it is far from trivial.

**Comment by absidue** [comment]
Reopening but will close as not feasible in 2 weeks time if someone doesn't provide an answer to the question above in that time.

**Comment by Atrate** [comment]
I see that Electron devs are quite unresponsive about the issue, despite wide community support for that feature (https://github.com/electron/electron/issues/8124). That sucks. Maybe FreeTube could implement a workaround given that it is a bit of a special case (hundreds of megabytes being written every hour or so, with videos being played).

**Comment by absidue** [comment]
@Atrate If the amount of writes are your concern, you can turn on the setting in the experimental settings section which disables Electron's network disk cache. It is still considered experimental as our custom image cache uses a lot more RAM and occasionally results in images not showing up, additionally as expected the lack of caching also means that it has to make more requests to YouTube and Invidious.

**Comment by absidue** [comment]
> If we could get some Cache controls, that would be awesome.

@Tharn The only controls are the command line flags that you mentioned.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> Reopening but will close as not feasible in 2 weeks time if someone doesn't provide an answer to the question above in that time.

Closing.
